File: Bee-Crowd/Iniciante/1074.py
def main():
  n = int(input())
  for i in range(n):
    x = int(input())
    par = x % 2 == 0
    if x == 0:
      print("NULL")
    elif x > 0 and par:
      print("EVEN POSITIVE")
    elif x > 0 and not par:
      print("ODD POSITIVE")
    elif x < 0 and par:
      print("EVEN NEGATIVE")
    elif x < 0 and not par:
      print("ODD NEGATIVE")
main()

File: Bee-Crowd/Iniciante/1074.c
#include <stdio.h>
int main(){
    int n, x;
    scanf("%d", &n);
    for(int i = 0; i < n; i++){
        scanf("%d", &x);
        int par = x % 2 == 0;
        if (x == 0){
            puts("NULL");
        } else if (x > 0 && par){
            puts("EVEN POSITIVE");
        } else if (x > 0 && !par){
            puts("ODD POSITIVE");
        } else if (x < 0 && par){
            puts("EVEN NEGATIVE");
        } else if (x < 0 && !par){
            puts("ODD NEGATIVE");
        }
    }
    return 0;
}